recipes/wdl-aid/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "WDL-AID" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: defdfcecb95c77d8ef3b7d04db878f430bd88019d159f710c8d6042361594b08
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - wdl-aid=wdl_aid.wdl_aid:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```

Comment:
It's okay for `noarch: python` packages to fail on Windows due to missing dependencies as they're built on Linux. If the missing package ever adds Windows support it should (🤞) just work.
```suggestion
```